tests/karate/src/test/java/fe/net/MockBackend.java

```diff
@@ -1,147 +0,0 @@
-package fe.net;
```

Comment:
I'm not sure I understand, given that MockBackend is removed, how are we sending messages to the frontend now?

Reply:
We use a real backend to reduce the complexity of the frontend tests and create e2e tests